deploy/openshift/template.yaml

```diff
@@ -1139,13 +1139,7 @@ objects:
                 - /opt/guac/guacgql
               args:
                 - --gql-backend
-                - ent
-                - --db-address
-                - postgres://$(DB_USER):$(DB_PASSWORD)@$(PGHOST):$(PGPORT)/$(DB_NAME)
-                - --db-driver
-                - postgres
-                - --db-debug
-                - 'true'
+                - inmem
```

Comment:
I think for "openshift" (AppSRE) we need to enable the database.

Reply:
This is for unblocking the stage deployment. Once RDS is configured in app-sre, we can set database: true in the openshift/values.yaml